src/AdvancedSet.ts

```diff
@@ -152,6 +152,16 @@ export default class AdvancedSet<T> {
     return this.toArray().every(test);
   }
 
+  public maxValue(setB: AdvancedSet<number>): number {
```

Comment:
Instead of finding the min of another set, could you find the min of the current set? So using `this.XYZ` instead of taking in a parameter